Recruit-Training/Advanced-Recruit-Training/src/BatteryStateOfChargeService.h

```diff
@@ -1,6 +1,7 @@
 #pragma once
 
 #include "I_BatteryStateOfChargeService.h"
+#include "BatteryData.h"
```

Comment:
When possible, it's best practice to include header files in the source, to reduce include bloat. 
The reason `I_BatteryStateOfChargeService.h` is included is because the header needs it to complete the inheritance.

https://softwareengineering.stackexchange.com/questions/167723/what-should-and-what-shouldnt-be-in-a-header-file